__tests__/app.test.js

```diff
@@ -183,3 +183,83 @@ describe("CORE: GET - /api/articles/:article_id", () => {
       });
     })
  
+
+
+
+   
+    
+
+describe("CORE: POST /api/articles/:article_id/comments", () => {
+  test("201 : Endpoint displays the inserted comment with correct properties/values", () => {
+    const input = {username : 'rogersop', body : 'big comment' }
+    return request(app)
+      .post("/api/articles/4/comments")
+      .send(input)
+      .expect(201)
+      .then(({body}) => {
+          expect(body).toHaveProperty("comment_id", expect.any(Number));
+          expect(body).toHaveProperty("votes", (0)); 
+          expect(body).toHaveProperty("created_at", expect.any(String)); 
+          expect(body).toHaveProperty("author", ('rogersop')); 
+          expect(body).toHaveProperty("body", ('big comment')); 
+          expect(body).toHaveProperty("article_id", (4)); 
+        });
```

Comment:
Could we also check that the comments are served most recent first and sorted accordingly from there?